cmd/honeyalb/main.go

```diff
@@ -228,3 +184,64 @@ Use '`+os.Args[0]+` --help' to see available flags.`)
 		os.Exit(1)
 	}
 }
+
+func ingestDist(sess *session.Session, id string, stater state.Stater, downloadsCh chan state.DownloadedObject) error {
```

Comment:
This is every similar for ALB, Cloudfront and ELB. A little refactoring would remove the duplicated code. Not doing it here because it is very low priority.